README.rst

```diff
@@ -1,11 +1,12 @@
 pycsob
 ======
 
-.. image:: https://circleci.com/gh/TwistoPayments/pycsob.svg?style=svg
-   :target: https://circleci.com/gh/TwistoPayments/pycsob
+.. image:: https://travis-ci.org/druids/pycsob.svg?branch=master
+    :target: https://travis-ci.org/druids/pycsob
```

Comment:
Says the repository is inactive.

Reply:
Yes, has to be activated after the merge.

Reply:
I think that would be necessary for the tests to run in the first place.

Reply:
![image](https://user-images.githubusercontent.com/424409/94197242-ccfeae80-feb5-11ea-81f8-3b8ea0f962ed.png)
 now is active

Reply:
Can you please amend the commit to see if Travis runs? Or is there anything else to be done to make it working? I still see Travis no output in this PR.

Reply:
I think it needs to be merged first to see it working here. But you can check the PR from my repository, it shows the checks there (I have it set up on my repository).

Reply:
OK, true.